src/Common/Nav.elm

```diff
@@ -0,0 +1,25 @@
+module Common.Nav exposing (navBar)
```

Comment:
@akabiru Yeah, there are. The navbar for the landing page is in `src/Langing/Nav.elm` while that of the app page is in `src/App/Common/Nav.elm`. I think we'd moved the file in question from `src/Common/` to `src/App/Common` when we refactored and broke the app into subviews. Unless I'm mistaken 😄. In which case I really need to go sleep. 😴